hackclub.com.yaml

```diff
@@ -2822,4 +2822,19 @@ internal:
 - ttl: 600
   type: A
   value: 104.131.182.116
+
```

Comment:
Hey! we don't allow personal sites on hackclub.com, please move this to dino.icu

Reply:
My bad! Changed this now!